>From 1723513e7c0828382ad5f1d58644db5dfbdb27a8 Mon Sep 17 00:00:00 2001
From: Steven McDonald <steven@steven-mcdonald.id.au>
Date: Tue, 13 May 2014 18:50:19 +1000
Subject: [PATCH] Document the default setting of Extension for the PackageDir
 driver

The PackageDir driver is documented as having configuration nearly
identical to the DirTree driver, but debconf.conf(5) does not mention
that the default value for Extension is different. This patch documents
that difference.
---
 doc/man/debconf.conf.5 | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/doc/man/debconf.conf.5 b/doc/man/debconf.conf.5
index 6b34d20..af691fb 100644
--- a/doc/man/debconf.conf.5
+++ b/doc/man/debconf.conf.5
@@ -200,10 +200,15 @@ room than the File database driver.
 .P
 This driver is configurable in the same ways as is the DirTree driver,
 plus:
+.RS
+.TP
+.B Extension
+An extension to add to the names of files. Defaults to "" (the empty string).
 .TP
 .B Mode
 The permissions to create files with. Defaults to 600, since the files could
 contain passwords in some circumstances.
+.RE
 .P
 As example stanza setting up a database using this driver:
 .P
-- 
2.0.0.rc2

